DashWallet/AppDelegate.m

```diff
@@ -96,6 +96,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [ExploreDashObjcWrapper configure];
     [CurrencyExchangerObjcWrapper startExchangeRateFetching];
     [CoinbaseObjcWrapper start];
+    [CrowdNodeObjcWrapper start];
```

Comment:
CrowdNode is initialized after the network is fully synced, which makes `reset` and `networkChange` observers not fire and not clear the `UserDefaults` if the wallet is reset during syncing.

Initializing it here takes care of that.